swift/tools/identify-environment.cmd

```diff
@@ -0,0 +1,6 @@
+@echo off
+SETLOCAL EnableDelayedExpansion
+
+echo { "swift": { "os": { "name": "macOS" } } }
+
+ENDLOCAL
```

Comment:
Can we drop this file as it's not used anywhere? Other than that, LGTM

Reply:
It will be used by the new `resolve build-environment` command when run on Windows, instructing the caller to use macOS to run the Swift autobuilder.

Reply:
Swift language pack is not available on Widows, so I don't see how it can be used on Windows.